app/assets/locales/android_translatable_strings.txt

```diff
@@ -381,6 +381,15 @@ notification.sync.connections.title=No Connection
 notification.sync.connections.detail=No active data connections
 notification.sync.connections.action=You don't have any active data connections configured. Try connecting your phone to wifi or turning mobile data on.
 
+
+connection.captive_portal.title=Captive Portal Detected
+connection.captive_portal.detail=Cannot access internet
+connection.captive_portal.action=You are in a captive portal. Try authenticating into captive portal using browser or turning mobile data on.
+
+connection.commcare_blocked.title=CommCare APIs blocked

```

Comment:
This doesn't really mean that CC is blocked but more possibly CC servers are down, the string "connection.task.commcare.html.fail" provides a better message for the suer in these scenarios.